CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## unreleased
+
+* Documented `Usage` and `Copylight and License Information` in README.md.
```

Comment:
As I was saying, we don't add a change log for documentation.

Reply:
OK